README.md

```diff
@@ -313,6 +313,14 @@ Depending on the specific model, ATtinys can have software only I2C, full hardwa
 
 Using ATTinyCore, I2Cwrapper firmware has been successfully tested on ATtiny85 (Digispark) and ATtiny88 (MH-ET-live) boards. Mileage with the available firmware modules may vary, though. Currently, only Pinl2C and TM1638liteI2C will run without changes. See the respective comment sections in the [Pin_Control.ino](https://github.com/ftjuh/I2Cwrapper/blob/main/examples/Pin_control/Pin_control.ino) and [TM1638lite.ino](https://github.com/ftjuh/I2Cwrapper/blob/main/examples/TM1638lite/TM1638lite.ino) examples for testing purposes. Of course, ATtinys are relatively slow and have limited memory. The firmware alone, without any modules enabled, currently uses 44% of a Digispark's usable 6586 bytes of flash memory, with the PinI2C module enabled it's 54%.
 
+### SAMD
+
+The most popular SAMD based systems are SAMD21 ("MO") and SAMD51 ("M4"). They come in lots of minor variations, but all seem to support hardware I2C, and the other usual stuff. 
```

Comment:
That's a bit vague, what's "usual stuff" and could it be there are samds without hardware I2C?

Reply:
Not the best wording. I've revised it.

I was hedging on the capabilities because I wasn't certain about the consistency across the chip family. I also had it mind that SAMD11 and other Cortex chips (like the M7 used in the Teensy) might work. I've been more explicit in the revision.

The SAMD's are ARM Cortex implementations, and as such, they are allowed many variations. Basic specs (RAM/EEPROM, clock, etc.), the number of pins, configuration of pins, protocols (I2S, CAN, etc.) does change between chips.

The SAMD21 spec sheet states "identical
peripheral modules" across the series. So, it sounds like I2C will always be there.

The SAMD51 spec sheet strongly implies I2C on the whole family.

refs
https://cdn.sparkfun.com/datasheets/Dev/Arduino/Boards/Atmel-42181-SAM-D21_Datasheet.pdf
https://ww1.microchip.com/downloads/aemDocuments/documents/MCU32/ProductDocuments/DataSheets/SAM_D5x_E5x_Family_Data_Sheet_DS60001507G.pdf